spec/features/borrower_dashboard_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "rails_helper"
+
+RSpec.feature "borrower dashboard" do
+  let!(:borrower) { User.create(email: "jeff@example.com",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.